repositories.bzl

```diff
@@ -31,12 +31,11 @@ def com_google_disco_to_proto3_converter_repositories():
     _maybe(
         jvm_maven_import_external,
         name = "google_java_format_all_deps",
-        artifact = "com.google.googlejavaformat:google-java-format:jar:all-deps:1.6",
+        artifact = "com.google.googlejavaformat:google-java-format:jar:all-deps:1.25.2",
```

Comment:
If this does not work, you can also try to use the formatter as a Maven plugin, see examples in [sdk-platform-java](https://github.com/googleapis/sdk-platform-java/blob/1eda55fc5e4cecedba3f8c163db0ce259d180a66/pom.xml#L31-L35), and their [Github page](https://github.com/spotify/fmt-maven-plugin).

Reply:
IIRC, some of the presubmit checks check the formatting, and I believe those run directly through Bazel....

Reply:
Trying the plugin, but the latest version is 2.25, which supports `google-java-format` 1.24.0 ([ref](https://github.com/spotify/fmt-maven-plugin/releases)). But I think I need `google-java-format` 1.25 (ideally 1.25.2) to properly support text blocks ([ref](https://github.com/google/google-java-format/releases/tag/v1.25.2)).